datafn/python/datafn/hooks.py

```
@@ -0,0 +1,69 @@
+        if env not in runs_on:
+            continue
+
+        hook_fn = getattr(plugin, hook_name, None)
+        if hook_fn is None:
+            continue
+
+        try:
+            result = await hook_fn(ctx, current)
+            if result is not None:
+                current = result
+        except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)